opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -418,8 +419,8 @@ protected void updateRedirectCode(ExtractableResponse<Response> response) {
     private void updateAvailableScas() {
         ExtractableResponse<Response> response = RestAssured
                 .given()
+                    .header(X_REQUEST_ID, DEFAULT_X_REQUEST_ID)
```

Comment:
Consent API calls should use generated request IDs (like it was before)

Reply:
random UUID is generated